bin/deploy

```diff
@@ -40,6 +40,7 @@ usage() {
 	    -v VERSION  a specific version of rubin-env conda package.
 	                By default will use the version specified in etc/settings.cfg.sh
 	    -r REF      git reference (SHA1) in scipipe_conda_env repository (backward compatibility)
+            -x TAG      eups tag to select the exact environment for that tag
```

Comment:
There seems to be a tab/space issue here.

I also think "exact" is a bit strong isn't it? Aren't we deriving the rubin-env but then installing whatever conda gives us for that version? We aren't installing the exact environment that was used to build the eups tag.

Reply:
No, this is installing the exact environment (via `--file` with `@EXPLICIT`).